contracts/token/ERC721/ERC721MetadataMintable.sol

```diff
@@ -8,7 +9,11 @@ import "../../access/roles/MinterRole.sol";
  * @title ERC721MetadataMintable
  * @dev ERC721 minting logic with metadata
  */
-contract ERC721MetadataMintable is ERC721, ERC721Metadata, MinterRole {
+contract ERC721MetadataMintable is Initializable, ERC721, ERC721Metadata, MinterRole {
+  function initialize() public initializer {
```

Comment:
Shouldn't `ERC721.initialize()` be called here?

Reply:
Just found it was added in https://github.com/OpenZeppelin/openzeppelin-zos/pull/24/commits/26fe5a57907c741f370bfe02e5f164fe7d855b99. Let's remove commit fb1fb0913c3e213067977d29b1ed15dfd644ece0 from this PR and merge, and discuss initializers on #24.

Reply:
Merging since this and #24 together leave this as we need it be.